microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -82,15 +82,19 @@ If Microsoft Edge detects typing in the Address bar that might contain sensitive
 
 
 <!-- ------------------------------ -->
 #### Deleting data
 
-Typed characters and the websites that you visit are stored locally on the device, per profile.  To delete that data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
+Typed characters and the websites that you visit are stored locally on the device, per profile.  To delete that data, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
 
-If Bing is your default search provider and you are signed in,<!-- todo: reformat paragraph as 3 list items starting here? --> you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  To clear your browsing history and delete websites from appearing as suggestions in the Address bar: select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**, and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.  You can delete the data that Microsoft collects from the Address bar and search suggestions features on Windows 10 and later: open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**, and then in **Delete diagnostic data**, select **Delete**.<!-- todo: is this sentence about Start menu for the correct feature?  seems to be about deleting Edge data from within the OS rather than from within Edge --><!-- old comment: deleting diagnostic data could use rewrite -->  All other data is deleted after 36 months.<!-- todo: what is "all other data"? -->
+If Bing is your default search provider and you are signed in,<!-- todo: reformat paragraph as 3 list items starting here? --> you can delete your searches by using the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).
+
+To clear your browsing history and delete websites from appearing as suggestions in the Address bar: select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Settings** > **Privacy, search, and services**,<!-- todo: update as separate groups of settings --> and then in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
```

Comment:
2nd instance:
did this UI end up in "Privacy and security" or "Search and services"? --
in the **Delete browsing data** section, click the **Choose what to clear** button; then in the **Delete browsing data** dialog, click the **Clear now** button.
@PatrizioBytes